coalib/results/Diff.py

```diff
@@ -278,16 +278,28 @@ def modified(self):
         return result
 
     @property
+    def modified(self):
+        """
+        Calculates the modified file, after applying the Diff to the original.
+
+        This property also adds linebreaks at the end of each line.
+        If no newline was present at the end of file before, this state will
+        be preserved, except if the last line is deleted.
+        """
+        return self._generate_linebreaks(self._raw_modified())
+
+    @property
     def unified_diff(self):
         """
         Generates a unified diff corresponding to this patch.
 
         Note that the unified diff is not deterministic and thus not suitable
         for equality comparison.
         """
+
```

Comment:
hmm let's be conform with everything else ;) so don't remove the newline.

Main problem is that those style changes are likely reverted again by someone else who's modifying this code. So we get many commits adding a newline, removing it again, adding it, and so on and so on.